easycwmp: fix ident for openwrt action script

Fix code identation, the code is hard enough to follow as is ;-)

Upstream-report: http://support.easycwmp.org/view.php?id=303

--- a/ext/openwrt/scripts/easycwmp.sh
+++ b/ext/openwrt/scripts/easycwmp.sh
@@ -316,14 +316,14 @@ handle_action() {
 					/sbin/uci import < $dwfile
 					fault_code="$?"
 				fi
-			if [ "$fault_code" != "0" ]; then
-					let fault_code=$E_DOWNLOAD_FAILURE+9000
-				common_json_output_fault "" "$fault_code"
-			else
-				$UCI_COMMIT
-				sync
-				reboot
-				common_json_output_status "1"
+				if [ "$fault_code" != "0" ]; then
+						let fault_code=$E_DOWNLOAD_FAILURE+9000
+					common_json_output_fault "" "$fault_code"
+				else
+					$UCI_COMMIT
+					sync
+					reboot
+					common_json_output_status "1"
 				fi
 			else
 				let fault_code=$E_DOWNLOAD_FAILURE+9000
@@ -336,12 +336,12 @@ handle_action() {
 			if [ "$dwfile" != "" ]; then
 				dwfile="$DOWNLOAD_DIR/$dwfile"
 				/sbin/sysupgrade $dwfile
-			fault_code="$?"
-			if [ "$fault_code" != "0" ]; then
-				let fault_code=$E_DOWNLOAD_FAIL_FILE_CORRUPTED+9000
-				common_json_output_fault "" "$fault_code"
-			else
-				common_json_output_status "1"
+				fault_code="$?"
+				if [ "$fault_code" != "0" ]; then
+					let fault_code=$E_DOWNLOAD_FAIL_FILE_CORRUPTED+9000
+					common_json_output_fault "" "$fault_code"
+				else
+					common_json_output_status "1"
 				fi
 			else
 				let fault_code=$E_DOWNLOAD_FAILURE+9000
